go.mod

```diff
@@ -1,16 +1,16 @@
 module github.com/filecoin-project/lassie
 
-go 1.20
+go 1.21
 
 require (
 	github.com/benbjohnson/clock v1.3.5
 	github.com/dustin/go-humanize v1.0.1
 	github.com/filecoin-project/go-data-transfer/v2 v2.0.0-rc7
 	github.com/filecoin-project/go-retrieval-types v1.2.0
 	github.com/filecoin-project/go-state-types v0.10.0
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.5.0
 	github.com/hannahhoward/go-pubsub v1.0.0
-	github.com/ipfs/boxo v0.15.1-0.20240125013539-09ff20c5abb6
```

Comment:
No, this can't be updated; maybe we need a note in here somehow, we maintain a branch with a fix in boxo that they refuse to integrate. This was accidentally updated in the last batch of bulk upgrades but reset again to a _new_ branch in https://github.com/filecoin-project/lassie/pull/465. If this is a strictly necessary upgrade then we'll have to do a new branch in boxo and put our fix in it and link that.

Reply:
not necessary. should be ok to fall back to v0.15.1-0.20240125013539-09ff20c5abb6